src/providers/twitch/TwitchChannel.cpp

```diff
@@ -37,6 +37,13 @@ namespace chatterino {
 namespace {
     constexpr int TITLE_REFRESH_PERIOD = 10;
     constexpr char MAGIC_MESSAGE_SUFFIX[] = u8" \U000E0000";
+    static const QString clipsLink("https://clips.twitch.tv/%1");
+    static const QString failureText =
+        "Failed to create a clip. Since this feature is new "
+        "and it requires extra scopes";
+    static const QString loginPromptText =
+        QString(" try adding your account again.");
+    static const auto accountsLink = Link(Link::OpenAccountsPage, QString());
```

Comment:
Sorry I just noticed this now (and it is kind of nitpicky but I will mention it nonetheless):
- Constants should be set in upper-case according to project conventions
- We use three different kind of initialization here: "canonical" `QString` constructor, implicit `QString` copy constructor and explicit `QString` copy constructor. I would prefer settling for one of these for consistency (I prefer the canonical version, as used for `clipsLink`.

Again, sorry I did not mention this in the first round of review

Reply:
Seems like I yolo copy-pasted the code and messed up kinds of initialization, will fix.